.vscode/launch.json

```diff
@@ -5,8 +5,7 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "address": "${config:mainframeAddress}",
-      "port": 9221,
+      "port": 9229,
```

Comment:
This works with the default inspect-brk in visual studio code, is anyone using this version?